lib/ansible/module_utils/crypto.py

```diff
@@ -46,3 +48,68 @@ def get_fingerprint(path):
         pass
 
     return fingerprint
+
+
+def load_privatekey(path, passphrase=None):
+    """Load the specified OpenSSL private key."""
+
+    try:
+        privatekey_content = open(path, 'r').read()
```

Comment:
'rb' might be more future proof here.  Current pyopenssl appears to allow both text and byte strings but in the documentation it looks like bytes may be preferred (the mirrored dump_* functions return byte strings).